org.eclipse.cdt.cross.arc.gnu/plugin.xml

```diff
@@ -679,7 +679,72 @@
             name="Use memory map from TCF"
             valueType="boolean">
       </option>
-
+      <option
+            applicabilityCalculator="com.arc.cdt.toolchain.arc.ArcApplicabilityCalculator"
+            defaultValue="false"
+            id="org.eclipse.cdt.cross.arc.gnu.base.option.target.includecdefines"
+            name="Include C defines from TCF"
+            valueType="boolean">
+      </option>
+      <option
+            applicabilityCalculator="com.arc.cdt.toolchain.arc.ArcApplicabilityCalculator"
+            id="org.eclipse.cdt.cross.arc.gnu.base.option.target.specsem"
```

Comment:
Name shouldn't include "target" which is for options that select processor specific extensions. "option.linker.specsem"?